deploy/k8s/dev/csidriver.yaml

```diff
@@ -0,0 +1,8 @@
+---
+
```

Comment:
nit: there are some stray newlines here and there

Reply:
Thanks for caching. Fixed